prow/cmd/gerrit/README.md

```diff
@@ -1,8 +1,8 @@
 # Gerrit
 
 Gerrit is a Prow-gerrit adapter for handling CI on gerrit workflows. It can poll gerrit
-changes from multiple gerrit instances, and trigger presubmits on Prow upon new patchsets on
-gerrit changes.
+changes from multiple gerrit instances, and trigger presubmits and postsubmits on Prow upon new
+patchsets on gerrit changes.
```

Comment:
bikeshed: something like `trigger presubmits upon new patchsets on gerrit changes, and postsubmits upon gerrit changes are merged`

Reply:
I didn't read carefully :)